tests/lib/rules/no-side-effects-in-computed-properties.js

```diff
@@ -446,6 +446,40 @@ ruleTester.run('no-side-effects-in-computed-properties', rule, {
         'Unexpected side effect in "test3" computed property.'
       ]
     },
+    {
+      // https://github.com/vuejs/eslint-plugin-vue/issues/1744
+      code: `app.component('test', {
+        computed: {
+          fooBar() {
+            this.$set(this, 'foo', 'lorem');
+            Vue.set(this, 'bar', 'ipsum');
+
+            this.$nextTick(() => {});
```

Comment:
I think `nextTick` is an issue that should be reported in the `no-async-in-computed-properties` rule.

https://eslint.vuejs.org/rules/no-async-in-computed-properties.html

What do you think?